ntbs-service/Models/SearchParameters.cs

```diff
@@ -11,5 +11,15 @@ public class SearchParameters
         
         [RegularExpression(@"[0-9]+", ErrorMessage = ValidationMessages.NumberFormat)]
         public string IdFilter { get; set; }
+
+        [MinLength(2)]
+        [RegularExpression(ValidationRegexes.CharacterValidation, ErrorMessage = ValidationMessages.StandardStringFormat)]
+        public string FamilyName { get; set; }
+
+        [MinLength(2)]
+        [RegularExpression(ValidationRegexes.CharacterValidation, ErrorMessage = ValidationMessages.StandardStringFormat)]
+        public string GivenName { get; set; }
+
+        public bool SearchParamsExist => !String.IsNullOrEmpty(GivenName) || !String.IsNullOrEmpty(FamilyName) || !String.IsNullOrEmpty(IdFilter);

```

Comment:
This is gonna get quire wordy.. Perhaps something like:
```
... => !(new List() {
GivenName, 
FamilyName, 
IdFilter}
).All(string.IsNullOrEmpty)
```
or similar?

Reply:
oh, this is nice, probably I will need something else as List should have .All method

Reply:
maybe `.Find(x => !string.IsNullOrEmpty(x)) != null` or smth else